hazelcast/src/main/java/com/hazelcast/jet/impl/connector/ReadJdbcP.java

```diff
@@ -191,4 +204,34 @@ private static Exception close(AutoCloseable closeable) {
         }
         return null;
     }
+
+    private static void setAutoCommitIfNecessary(Connection connection, Properties properties) {
+        try {
+            String key = JdbcPropertyKeys.AUTO_COMMIT;
+            if (properties.containsKey(key)) {
+                String value = properties.getProperty(key);
+
+                if (isBoolean(value)) {
+                    boolean autoCommit = Boolean.parseBoolean(value);
+                    connection.setAutoCommit(autoCommit);
+                }
```

Comment:
IMHO there should be `else` section, that throws an exception - incorrect configuration value.